packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1194,6 +1220,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private formMetadata(): IContainerRuntimeMetadata {
         return {
+            ...this.createContainerMetadata,
             summaryFormatVersion: 1,
             disableIsolatedChannels: this.disableIsolatedChannels || undefined,
```

Comment:
this.disableIsolatedChannels is boolean, curious why we have || undefined here?

Reply:
maybe because this flag is new and doesn't exist for old files?

Reply:
it is also listed at optional